website/addons/base/views.py

```diff
@@ -435,6 +435,7 @@ def addon_view_file(auth, node, node_addon, guid_file, extras):
         # Note: must be called after get_or_start_render. This is really only for github
         'size': size,
         'extra': json.dumps(getattr(guid_file, 'extra', {})),
+        'view_url': guid_file.view_url,
```

Comment:
This should probably go in the `urls` dict above. Also, `view_url` is ambiguous (could be confused with the URL to view the file on OSF)--how about `external`?